editor/editor_node.h

```diff
@@ -125,6 +126,13 @@ class EditorNode : public Node {
 		EDITOR_ASSETLIB
 	};
 
+	// A copy of the name casing used for scripts, modified *barely* to be used with scenes
```

Comment:
```suggestion
	// A copy of the name casing used for scripts, modified *barely* to be used with scenes.
```
That comment is not necessary tbh.